admin/common_auth/forms.py

```diff
@@ -15,3 +16,11 @@ def __init__(self, *args, **kwargs):
         super(CustomUserRegistrationForm, self).__init__(*args, **kwargs)
         self.fields['first_name'].required = True
         self.fields['last_name'].required = True
+
+    def clean_password1(self):
```

Comment:
@Ghalko Chris enlightened me on model-level character limits, so I will use that for password restriction instead of a form-level validation